hack/common.sh

```diff
@@ -50,8 +50,7 @@ build_WMCO() {
       error-exit "OPERATOR_IMAGE not set"
   fi
 
-  $OSDK build "$OPERATOR_IMAGE" --image-builder $CONTAINER_TOOL $noCache \
-    --go-build-args "-ldflags -X=github.com/openshift/windows-machine-config-operator/version.Version=${VERSION}"
+  $CONTAINER_TOOL build . -t "$OPERATOR_IMAGE" -f build/Dockerfile $noCache
```

Comment:
Does this mean that `operator-sdk build` will not work now? If that is the case, you also need to update [HACKING.md](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/HACKING.md). If `operator-sdk build` works then you don't need to update the doc.

Reply:
it should not break `operator-sdk build` command and it will work. But now that we are removing it, i think we should remove it from doc as well.

Reply:
Just realized you have not updated the doc.